images/node/Dockerfile

```diff
@@ -10,8 +10,6 @@
 #
 FROM openshift/origin
 
-COPY scripts/* /usr/local/bin/
-COPY system-container/system-container-wrapper.sh /usr/local/bin/
```

Comment:
Are we certain that we want the system container stuff in the RPMs? I thought we were going to ship that in some other manner /cc @tdawson